packages/components/src/Button/Button.css

```diff
@@ -1,16 +1,12 @@
-:root {
+.button {
```

Comment:
also see some overrides in product for `--button--` variables`

probably fine, but worth verifying quick

Reply:
Interesting, this one seems to have the most examples so I might kick this one out into its' own PR. A lot of the usages are people seemingly attempting to rebuild an Atlantis button from scratch, which is theoretically as simple as dropping in the semantic tokens they actually, but there are a few explicit overrides that appear to be happening.

Reply:
OK changed the usages [in Jobber](https://github.com/GetJobber/Jobber/pull/48119) - this will need a tweak to the one remaining usage of these public values to consume from `--public--` but I think that's safe to do on the import of these changes?